ObjectiveZipLib/Objective-Zip/ZipProgressBase.mm

```diff
@@ -45,7 +45,18 @@ - (id) initWithZipFile:(NSURL *)zipFileURL
 
 - (void)dealloc
 {
-   [self performZipToolCleanup];
+   @try
+   {
+      [self performZipToolCleanup];
+   }
+   @catch ( NSException * e )
+   {
+      NSLog(@"Exception thrown deallocing ZipProgressBase %@", e);
+   }
+   @catch ( id )
```

Comment:
why not print out the id ?

Reply:
I am not 100% sure that ids can always be printed.